packages/devextreme/js/ui/validation_engine.js

```diff
@@ -763,7 +768,7 @@ const ValidationEngine = {
 
     _shouldRemoveGroup(group, validatorsInGroup) {
         const isDefaultGroup = group === undefined;
-        const isValidationGroupInstance = group && group.NAME === 'dxValidationGroup';
+        const isValidationGroupInstance = group && validGroupNames.includes(group.NAME);
```

Comment:
[naming]
Previously `isValidationGroupInstance ` was a good name because this variable stored a boolean value which pointed if it's a validation group instance or not.
But after your change this naming is not actual because this variable now can be true even if it's not a validation group instance (in case when it's a Form instance).
```suggestion
        const isNonRemovableGroup = group && validGroupNames.includes(group.NAME);
```
P.s. I write this just as a feedback for you to not repeat it in the future